.codecov.yml

```diff
@@ -7,7 +7,16 @@ codecov:
 ignore:
   - "cron/**/*"
   - "clients/mockclients/**/*"
-  - "attestor/command/**/*"
+  # ignoring them as these are part of the CLI tool.
+  - "attestor/command/check.go"
+  - "attestor/command/cli.go"
+  - "attestor/command/sign.go"
+  - "attestor/root.go"
+  # ignoring them as these are internal tools for generating docs.
+  - "docs"
```

Comment:
It seems the pattern to ignore a folder should be:
"docs/**/*"

https://docs.codecov.com/docs/ignoring-paths

Reply:
👍 Thanks